frontend/src/routes/(authenticated)/admin/EditUserAccount.svelte

```diff
@@ -111,11 +109,10 @@
     userIsLocked = data!.setUserLocked.user!.locked;
   }
   let form = $derived(formModal?.form());
-  // This is a bit of a hack to make sure that the email field is not required if the user has no email
-  // even if the user edited the email field
-  run(() => {
-    if (form && $form && !$form.email && _user && !_user.email) $form.email = null;
-  });
+  // TODO: We used to have a hack here to make sure that the email field is not required if the user
```

Comment:
Would this work? I just added `&& $form.email !== null`.
```
$effect(() => {
    if (form && $form && !$form.email && $form.email !== null && _user && !_user.email) $form.email = null;
  });
```

Reply:
Yep, that works. Fixed in commit ed86cda2.